python/py-pytest-timeout/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        pytest-dev pytest-timeout 1.4.2
+revision            0
+name                py-${github.project}
+fetch.type          git
```

Comment:
Why `fetch.type git` if you have checksums present?

Reply:
Thanks, forgot to remove that after discovering that this build wants a clone.